overlord/snapstate/snapstate.go

```diff
@@ -146,22 +185,44 @@ func Remove(s *state.State, snapSpec string, flags snappy.RemoveFlags) (*state.T
 	}
 
 	// trigger remove
-	unlink := s.NewTask("unlink-snap", fmt.Sprintf(i18n.G("Deactivating %q"), snapSpec))
-	unlink.Set("snap-setup", ss)
+
+	// last task but the one holding snap-setup
+	forget := s.NewTask("forget-snap", fmt.Sprintf(i18n.G("Forgetting removed revision of %q"), snapSpec))
```

Comment:
s/Forgetting/Forget/

Do we really need this task, though? Shouldn't that be done inside remove-snap-files, which is the true "forget it"?